README.md

```diff
@@ -43,7 +52,7 @@ Here are some Firebase build workarounds that you may need during normal develop
 No tests found. This usually means that your test classes are not in the form that your test runner expects (e.g. don't inherit from TestCase or lack @Test annotations).
 ```
 
-If you'd like to test the fully functional Firebase build, you should first setup the environment variables following these [instructions](https://github.com/mozilla-tw/Rocket/blob/4fedf245c4382122283ca8ec701a5ff18c9bf779/app/build.gradle#L346) and select **focusWebkitFirebase**
+2. If you'd like to test the fully functional Firebase build, you should first setup the environment variables following these [instructions](https://github.com/mozilla-tw/Rocket/blob/4fedf245c4382122283ca8ec701a5ff18c9bf779/app/build.gradle#L346) and select **focusWebkitFirebase**
```

Comment:
I don't treat it as the same level as some Firebase workaround cause this line is about using a Firebase build. However, I have no concern if this looks more reasonable to you.